resources/deployments/controller/controller_deployments.go

```diff
@@ -297,6 +298,24 @@ func ParseLookupQuery(vals url.Values) (deployments.Query, error) {
 		query.SearchText = search
 	}
 
+	createdBefore := vals.Get("created_before")
+	if createdBefore != "" {
+		if createdBeforeTime, err := parseEpochToTimestamp(createdBefore); err != nil {
+			return query, errors.Wrap(err, "timestamp parsing failed")
```

Comment:
include which query param are you parsing in error message, if you use both in request you'll need to know which is wrong